src/ext/store.ts

```diff
@@ -53,6 +55,18 @@ export const derivedStateProperties: Record<string, DerivedStateProperty> = {
       "The geometry of the currently selected place in GeoJSON format.",
     selector: selectedPlaceGeometrySelector,
   },
+  selectedPlaceId: {
+    type: "str | None",
+    description:
+      "The identifier of the currently selected place.",
+    selector: selectedPlaceIdSelector,
+  },
+  selectedPlaceGroup: {
+    type: "dict[str, Any] | None",
```

Comment:
Shouldn't the type here be the following @forman ?
I did not find it returning None as it only returns `PlaceGroup[]`
```suggestion
    type: "list[dict[str, Any]]",
```

Reply:
Yepp.